docker-amundsen-atlas.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+services:
+  atlas:
+      image: wbaa/rokku-dev-apache-atlas:0.0.8
+      container_name: atlas_amundsen
+      ports:
+          - 21000:21000
+      networks:
+        - amundsennet
+  amundsensearch:
+      image: seanazoci/amundsen-search:latest
+      container_name: amundsensearch
+      ports:
+        - 5001:5000
+      #depends_on:
+      #  - elasticsearch
+      networks:
+        - amundsennet
+  amundsenmetadata:
+      image: amundsen-metadata:latest
+      container_name: amundsenmetadata
+      depends_on:
+        - atlas
+      ports:
+        - 5002:5000
+      networks:
+        - amundsennet
+      environment:
+        - CREDENTIALS_PROXY_USER=admin
+        - CREDENTIALS_PROXY_PASSWORD=admin
+        - PROXY_HOST=atlas
+        - PROXY_PORT=21000
+        - PROXY_CLIENT=ATLAS
```

Comment:
this should be `metadata_service.proxy.atlas_proxy.AtlasProxy` i.e., the class of the Proxy Client.

Reply:
I use the lookup table, so this can be either NEO4J or ATLAS at the moment (capitalization matters)